schemas/portals.json

```diff
@@ -143,7 +143,6 @@
                 "title": "SD-WAN Portal version",
                 "description": "Portal version to be pulled from Docker hub",
                 "default": "",
-                "sectionEnd": "Portal",
```

Comment:
Why was this removed?

Reply:
I forgot to add that back. Initially, I had added a couple of other parameters in the schema but removed them later on. I'll add the "sectionEnd" back